packages/askui-nodejs/src/utils/transformations.ts

```diff
@@ -40,21 +40,26 @@ export async function resizeBase64ImageWithSameRatio(
   }
   try {
     let resizeRatio = 1;
-    const orignalImage = await read(Buffer.from(base64ImageString.replace(/^data:image\/png;base64,/, ''), 'base64'));
-    const imageHeight = orignalImage.getHeight();
-    const imageWidth = orignalImage.getWidth();
+    const orignalImage = sharp(Buffer.from(base64ImageString.replace(/^data:image\/png;base64,/, ''), 'base64'));
```

Comment:
Can we encapsulate the adding (line 12, 38, 62 etc.) and removing of the prefix, e.g., inside class?